spec/orders.json

```diff
@@ -1139,76 +1487,31 @@
         },
         "misc": {
           "type": "object",
-          "properties": { "data": { "type": "object", "properties": {} } },
+          "required": [
+            "data"
+          ],
+          "properties": {
+            "data": {
+              "type": "object",
+              "properties": {}
+            }
+          },
           "description": "Miscellaneous data for the order. For example, analytics data."
+        },
+        "cart_id": {
+          "type": "string",
+          "description": "Unique identifier for the cart",
+          "example": "cjzyvxtm207z4rns56tif8job"
         }
       }
     },
-    "universalOrderResult": {
+    "orderOutputResult": {
```

Comment:
@KameronSturgess @haythamlabrini Can you please check the monorepo to see whether this tracks. I changed the output schema in `no-hassle` to return orderOutputSchema, but since we don't have proper integration tests checking this actually gets returned, I checked Postman to see the response. However, it feels a bit concerning our docs aren't returning these properties anymore. Either we need to update the `orderOutputSchema` or our docs were wrong for quite a while